app/views/checklist_tests/_checklist_measure.html.erb

```diff
@@ -72,13 +75,38 @@
                           <i class = 'fa fa-fw fa-times text-danger' aria-hidden="true"></i>
                         <% end %>
                         <%= criteria_field.text_field :code, hide_label: false %>
+                          <div id="lookupModal" class="modal fade" role="dialog">
```

Comment:
This view is getting pretty bloated. I think it would be best to separate out the modal into a partial.